Zeeguu Reader/Zeeguu Reader/ArticleViewController.swift

```diff
@@ -42,16 +44,23 @@ class ArticleViewController: UIViewController {
 		self.view.backgroundColor = UIColor.whiteColor()
 		if let art = article {
 			let sv = UIScrollView.autoLayoutCapapble()
-			let view = ArticleView(article: art)
-			let views: [String: AnyObject] = ["sv": sv,"v":view, "top":self.topLayoutGuide]
+			articleView = ArticleView(article: art, delegate: self)
+			let views: [String: AnyObject] = ["sv": sv, "v": articleView!]
+			
+			sv.addSubview(refresher)
 			
 			self.view.addSubview(sv)
 			self.view.addConstraints(NSLayoutConstraint.constraintsWithVisualFormat("H:|[sv]|", options: NSLayoutFormatOptions(rawValue: 0), metrics: nil, views: views))
 			self.view.addConstraints(NSLayoutConstraint.constraintsWithVisualFormat("V:|[sv]|", options: NSLayoutFormatOptions(rawValue: 0), metrics: nil, views: views))
 			
-			sv.addSubview(view)
+			sv.addSubview(articleView!)
 			self.view.addConstraints(NSLayoutConstraint.constraintsWithVisualFormat("H:|[v(==sv)]|", options: NSLayoutFormatOptions(rawValue: 0), metrics: nil, views: views))
 			self.view.addConstraints(NSLayoutConstraint.constraintsWithVisualFormat("V:|[v]|", options: NSLayoutFormatOptions(rawValue: 0), metrics: nil, views: views))
+			
+			refresher.beginRefreshing()
+			
+			let translateBut = UIBarButtonItem(title: "TRANSLATION_MODE".localized, style: .Plain, target: self, action: "toggleTranslationMode:")
```

Comment:
Agreed, but where to display that? Especially on iPhone, there is not much space, so how would you suggest we display it?